From a93ee5d433308d2cac76ac9502e34c953ff96337 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Fri, 28 Dec 2018 09:33:07 -0500
Subject: [PATCH 2/4] Ignore "no $pid_file configured, not checking it" lines.

When no PID file is configured (for example, when amavisd is running
in the foreground), this message is logged. We already ignore the
other types of PID file notifications, so ignore this one too.
---
 amavis-logwatch | 1 +
 1 file changed, 1 insertion(+)

diff --git a/amavis-logwatch b/amavis-logwatch
index ffb502b..6d06793 100644
--- a/amavis-logwatch
+++ b/amavis-logwatch
@@ -1958,6 +1958,7 @@ sub create_ignore_list() {
    push @ignore_list_final, qr/^fish_out_ip_from_received: /;
    push @ignore_list_final, qr/^Waiting for the process \S+ to terminate/;
    push @ignore_list_final, qr/^Valid PID file \(younger than sys uptime/;
+   push @ignore_list_final, qr/^no \$pid_file configured, not checking it/;
    push @ignore_list_final, qr/^Sending SIG\S+ to amavisd/;
    push @ignore_list_final, qr/^Can't send SIG\S+ to process/;
    push @ignore_list_final, qr/^killing process/;
-- 
2.19.2

